### PR TITLE
[Move Expando part 4]Lps 190974

### PR DESCRIPTION
**Pull request by slnn**
https://liferay.atlassian.net/browse/LPS-190974

### CODE REVIEW

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/expando/expando-test/src/com/liferay/expando/test/ExpandoConverterUtilTest.java [hunk 1]
```diff
@@ -12,10 +12,10 @@
  * details.
  */
 
-package com.liferay.portlet.expando.util;
+package com.liferay.expando.test;
```
Comment (dantewang) [inline modules/apps/expando/expando-test/src/com/liferay/expando/test/ExpandoConverterUtilTest.java:15]: @slnn The directory structure is wrong. A unit test should be in `src/test/java/....`. You missed the `test/java` part.

Comment (dantewang) [reply modules/apps/expando/expando-test/src/com/liferay/expando/test/ExpandoConverterUtilTest.java:15]: @slnn Also, unit tests are usually put in the `src/test/java` in the same module of the tested class. So you can put this test in `expando-api`.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/expando/expando-test/src/com/liferay/expando/test/ExpandoConverterUtilTest.java (hunk 1 above)
Comment (slnn) [reply modules/apps/expando/expando-test/src/com/liferay/expando/test/ExpandoConverterUtilTest.java]: Done

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
ci:test:sf

**Review by dantewang — APPROVED** [review]

**Comment by catarineaguiar** [comment]
no regressions found

**Review by dantewang — APPROVED** [review]

**Comment by tinatian** [comment]
@slnn Please rebase.

Thanks.
Tina

**Comment by slnn** [comment]
Hi @tinatian 

I have rebased and updated this pull, no logic change.


Regards
Lily

**Comment by dantewang** [comment]
ci:test:sf

**Comment by slnn** [comment]
Update with the new CopyrightText

**Comment by slnn** [comment]
ci:test:sf

**Comment by slnn** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by zhangyan0701** [comment]
ci:test:ldap

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by tinatian** [comment]
ci:forward:force

**Pull request closed without merging**